src/components/Card/Card.scss

```diff
@@ -0,0 +1,26 @@
+[data-priority='1'] {
```

Comment:
In general we don't use `data-*` attributes for styling. Rather we would use a classname or id to target individual elements.